composer.json

```diff
@@ -46,8 +46,7 @@
         "fzaninotto/faker": "^1.5.0",
         "infection/infection": "^0.6.0",
         "mockery/mockery": "1.0.*@dev",
-        "phpunit/phpunit": "^6.4.4",
-        "phpunit/phpunit-mock-objects": "^4.0"
```

Comment:
Removing this requirement as `phpunit/phpunit:^6.5.0` depends on `phpunit/phpunit-mock-objects:^5.0.0`.

Reply:
Don't we use the ```phpunit/phpunit-mock-objects``` directly in our tests?
Because if so i'd rather have it explicitly in our own dependencies.

Reply:
It's a direct dependency of `phpunit/phpunit`, as can be seen in [`composer.json`](https://github.com/sebastianbergmann/phpunit/blob/6.5.1/composer.json#L38), so that should be fine.

Reply:
Since this is more of a personal taste thing, what do you think @chartjes @mdwheele ?

Reply:
I'm fine with it. I'm pretty sure if `phpunit/phpunit-mock-objects` is every removed from `phpunit/phpunit`'s dependencies, it'll happen in a major. It's been a composer dependency since 4.0 (and likely before, but Composer didn't exist).